utils/vlog/tests/fig28-lutff-pair/golden.pb_type.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='utf-8'?>
+<pb_type xmlns:xi="http://www.w3.org/2001/XInclude" name="PAIR" num_pb="1">
+  <clock name="CLK" num_pins="1"/>
+  <input name="I" num_pins="4"/>
+  <output name="O" num_pins="1"/>
+  <pb_type blif_model=".subckt dff" name="ff" num_pb="1">
+    <xi:include href="ff/ff.pb_type.xml" xpointer="xpointer(pb_type/child::node())"/>
+  </pb_type>
+  <pb_type blif_model=".subckt lut4" name="lut" num_pb="1">
```

Comment:
```suggestion
  <pb_type blif_model=".subckt LUT4" name="lut" num_pb="1">
```

Reply:
```suggestion
  <pb_type name="lut" num_pb="1">
```
`blif_model` should be a `<blif_model>` tag inside the `lut/lut.pb_type.xml`.

Reply:
fixed